docs/extend/menu.md

```diff
@@ -1,22 +1,20 @@
 <!--[metadata]>
 +++
-title = "Extend Engine"
+title = "Extend Docker"
 description = "How to extend Docker Engine with plugins"
 keywords = ["extend, plugins, docker, documentation, developer"]
+type="menu"
 [menu.main]
 identifier = "engine_extend"
-parent = "engine_use"
-weight = 6
+parent="engine_use"
+weight = 0
 +++
 <![end-metadata]-->
 
 
-## Extending Docker Engine
+## New Docker Plugin System
```

Comment:
I don't like the use of "New" but in this case it doesn't matter because that is in `menu.md` that never renders because it should be `type="menu"`